src/Packetery/Module/Checkout/SessionService.php

```diff
@@ -33,6 +33,10 @@ public function getChosenMethodFromSession(): string {
 			}
 		}
 
+		if ( $chosenShippingRate === false ) {
```

Comment:
Nedává mi smysl nastavovat `$chosenShippingRate` na false. Bylo by myslím lepší řešit rovnou na řádku 31.